src/game/client/optionpreferences.cpp

```diff
@@ -463,7 +463,13 @@ uint16_t OptionPreferences::Get_Firewall_Port_Override()
         return g_theWriteableGlobalData->m_firewallPortOverrides;
     }
 
-    return atoi(it->second.Str());
+    int ret = atoi(it->second.Str());
+
+    if (ret >= 65536) {
```

Comment:
original is if ( v11 < 0 || v11 > 0xFFFF )

Reply:
fixed.